middleware/authentication/authentication.test.js

```diff
@@ -0,0 +1,57 @@
+/* eslint-env jest */
+// NPM dependencies
+jest.mock('basic-auth')
+const basicAuth = require('basic-auth')
+
+// Local dependencies
+const authentication = require('./authentication.js')
+
+// Local variables
+const userDetails = { name: 'secret-username', password: 'secure-password' }
+const next = jest.fn()
+const res = {send: jest.fn(),
```

Comment:
This line of code has been superseded by another commit. I changed the approach because I found that as I was adding more tests I needed more methods and instead of manual mocking out all the response method it would be good to just use the Express response object